extensions/wikia/ArticleComments/classes/ArticleCommentList.class.php

```diff
@@ -408,11 +408,18 @@ public function getData( $page = 1 ) {
 		$this->preloadFirstRevId( $comments );
 		$pagination = $this->doPagination( $countComments, count( $comments ), $page );
 
+		$commentTitle = Title::makeTitle(
+			MWNamespace::getTalk( $this->mTitle->getNamespace() ),
+			$this->mTitle->getDBkey() . '/' . ARTICLECOMMENT_PREFIX
+		);
+
+		$canComment = $commentTitle->userCan( 'create' );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$canComment" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)